index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portifolio</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <header class="toolbar">
+        <a href="index.html" class="name"> SOLOMON</a>
+        <nav>
+          <img class="huberger" src="/assets/Union.png" alt="Dropdown menu">
+        </nav>
+    </header>    
+     <section class="first-page">
+           <div class="headline">
+             <h1 class="hey">Hey There  I'm Solomon</h1>
+             <h2 class="softwaredev"> I am software developer</h2>
+             <p class="headline-p">I can help you build a product , feature or website Look through some of my work
+                 and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+             </p>
+          </div>
+          <ul class="socila-link">
+            <li class="social"><a href="https://github.com/solog0039"><img  src="./assets/socialmedia/github.png" alt="Github"></a></li>
+            <li class="social"><a href="www.linkedin.com/in/solomon-kidanu-62a994232"><img  src="./assets/socialmedia/linkidin.png" alt="linkedin"></a></li>
+            <li class="social"><a href="##"><img  src="./assets/socialmedia/tump.png" alt="tum"></a></li>
+            <li class="social"><a href="https://twitter.com/Solomon57320119"><img  src="./assets/socialmedia/tiwtter.png" alt="tiwtter"></a></li>
+            <li class="social"><a href="##"><img  src="./assets/socialmedia/m.png" alt="M"></a></li>
+          </ul>
+</section>
+<section class="cards">
+   <div class="h21-line">
+      <h1 class="h21">My Recent Works</h1>
+      <hr class="line">
+   </div>
+     <img class="main-image" src="assets/images/Placeholder.png" alt="main image">
+     
+     <div class="card-1">
+        <h2 class="multi-story">Multi-Post Stories</h2>
+        <p class="p21">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
```

Comment:
- [ ] Kindly adjust the font family of this section to match the original design. Also, adjust the text color of the `see project` button.

![image](https://user-images.githubusercontent.com/98453979/213496706-0acab91b-e5fd-4271-be49-66ed766cc4e7.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/213496910-e5cf384d-2389-47db-80f8-7d3f7ffb77ec.png)

- [ ] Kindly adjust center the `html` text on your project cards
 
![image](https://user-images.githubusercontent.com/98453979/213497573-5f1088c6-72ab-4bce-ab3c-307855f87b9b.png)